server/views/azuread-b2c-login.html

```diff
@@ -12,6 +12,8 @@
       href="{{hostname}}/assets/images/favicon.ico"
       type="image/x-icon"
     />
+    <script src="{{hostname}}/assets/javascripts/errors.js"></script>
+    <script src="{{hostname}}/assets/javascripts/xhr.js"></script>
     <link rel="mask-icon" href="{{hostname}}/assets/images/govuk-mask-icon.svg" color="#0b0c0c" />
```

Comment:
Are these files there twice now?

Reply:
Sorry forgot to change branch and ended up wit conflicts, missed this one 🤦